host-configs/lassen-blueos_3_ppc64le_ib_p9-clang@upstream_gfortran-cuda.cmake

```diff
@@ -57,9 +57,9 @@ set(MPIEXEC_EXECUTABLE "/usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-rele
 #---------------------------------------
 # Library Dependencies
 #---------------------------------------
-set(TPL_ROOT "/usr/WS2/smithdev/libs/blueos_3_ppc64le_ib_p9/2020_10_19_15_04_12/clang-upstream_gfortran" CACHE PATH "")
+set(TPL_ROOT "/usr/WS2/smithdev/libs/blueos_3_ppc64le_ib_p9/2020_10_27_08_37_57/clang-upstream_gfortran" CACHE PATH "")
 
-set(AXOM_DIR "${TPL_ROOT}/axom-0.4.0serac" CACHE PATH "")
+set(AXOM_DIR "${TPL_ROOT}/axom-0.4.0p1" CACHE PATH "")
```

Comment:
was this reverted on purpose?

Reply:
It was actually an oversight when I was juggling images around - the change to our version of axom's package.py is actually in https://github.com/LLNL/serac/pull/88/files#diff-f2386102a0ae90f37dd97db98adcfd7e856a8bf99811ffe7649e55245e2368e9R58 (which hasn't been merged).

That PR needs to be merged in before this one, so there will be a rebuild with the correct tag name.